README.md

```diff
@@ -31,249 +31,23 @@ boilerplate code as new improvements are discovered.
-already hooked up to redux using the `@connect` middleware.
-
-Example:
-```
-gluestick generate container MyContainer -E shared
-```
-
-#### Component Generator
-The component generator will create a basic React component and a starting test
-file for that component.
-
-Example:
-```
-gluestick generate component MyComponent
-```
-
-#### Reducer Generator
-The reducer generator will create a new base redux reducer for you and
-automatically export it from the `reducers/index.js` file.
-
-Example:
-```
-gluestick generate reducer todos -E shared
-```
-
-### Destroy command
-
-You can also use `destroy` to delete generated files: `component`, `container`
-and `reducer`.
-
-Example:
-```
-gluestick destroy component MyComponent -E apps/some-app
-```
-
-## Code quality
-
-### Tests
-GlueStick sets up a testing environment using [Jest](https://facebook.github.io/jest/) and [Enzyme](https://github.com/airbnb/enzyme).
-
-You simply need to create files in any `__tests__` directory with the extension `.test.js` and they
-will be executed through the test runner using:
-```
-npm test
-```
-
-Tests can also be typed using `Flow` (see below). We use [flow-typed](https://github.com/flowtype/flow-typed) for Jest under the hood.
-
-### Types
-All generated files will use `/* flow */` and will be typed.
-We recommend you to do the same for your own files. To run flow just run:
-```
-npm run flow
-```
-
-### Linting
-There is a predefined linting configuration (following [airbnb config](https://github.com/airbnb/javascript/tree/master/packages/eslint-config-airbnb))
-for your project. Run it using:
-```
-npm run lint
-```
-
-### Integration
-All of the above code quality commands can be executed simultaneously by:
-```
-npm run ci
-```
-
-## Styles
-The preferred way to style components in the GlueStick environment is to use
-[Radium](https://github.com/FormidableLabs/radium), but we also actively use
-and support css and sass.
-
-#### CSS and SASS
-If you want to include a base stylesheet (like bootstrap css), you can import
-your stylesheet at the top of any file and it will be included when the page
-loads. Any updates to your styles will also be hot loaded. You can use plain
-css or sass.
-
-Example:
-Edit /Index.js
-```
-import React, { Component } from "react";
-import "assets/css/my-custom-css.css";
-…
-```
-
-The code above will automatically extract the css from the referenced file and
-include it in a base css file that is included on new page loads. References to
-images and font files will automatically be handled as well.
-
-_If you would like to see better css support, please submit a pull request :)_
-
-## Caching
-Caching support has been added for server side rendering. This lets you cache
-entire page responses on pages where it makes sense like a home or landing
-page.
-
-Example:
-```
-<Route path="/" component={HomeApp} cache={true} />
-```
-
-*Route property:*
-`cache` - boolean
+* [Commands](docs/COMMANDS.md)
```

Comment:
I miss a `Getting started` link as the first link. For me it would contain the commands that the end users will use: new, start, start-client, start-server, generate (which includes destroy).

Then I'd have a section about Tests, Flow and Eslint. And a development section with less important commands like `gluestick reinstall-dev`.

More or less, separating it into basics (usage) and advance (to develop, test, etc), similar to https://github.com/ReactTraining/react-router/tree/master/docs